config/config.exs

```diff
@@ -119,8 +119,28 @@ config :trento, Trento.Scheduler,
   debug_logging: false
 
 config :trento, Trento.Integration.Telemetry, adapter: Trento.Integration.Telemetry.Suse
-config :trento, Trento.Integration.Checks, adapter: Trento.Integration.Checks.Runner
-config :trento, Trento.Clusters, checks_adapter: Trento.Clusters.Runner
+
+config :trento, :messaging, adapter: Trento.Messaging.Adapters.AMQP
+config :trento, Trento.Integration.Checks, adapter: Trento.Integration.Checks.Wanda
+config :trento, Trento.Messaging.Publisher, adapter: Trento.Messaging.Adapters.AMQP
+config :trento, Trento.Clusters, checks_adapter: Trento.Clusters.Wanda
+
+config :trento, Trento.Messaging.Adapters.AMQP,
+  publisher: [
+    exchange: "trento.checks",
+    connection: "amqp://wanda:wanda@localhost:5674"
+  ]
+
+config :trento, Trento.Integration.Checks.Wanda.Messaging.AMQP,
```

Comment:
I Think we still need the wanda configuration. 
I see it this way:
- dev config (uses the local docker compose forwarded port)
- wanda config (it's a dev config that connects to wanda docker compose forwarded port)

By doing this we can develop web alongside wanda or standalone.

Reply:
I don't really know a current use case where I would like to have the web running without wanda, but there might be some...
Anyway, it's true that with the current change, the current docker-compose wouldn't have much sense.
I will add back the wanda config

Reply:
Done!

Reply:
the idea is to start web "standalone" when developing. not every time you need wanda to write code on trento web.